src/multibody/model.hpp

```diff
@@ -292,43 +292,8 @@ namespace se3
     ///
     bool existFrame (const std::string & name, const FrameType& type = (FrameType) (JOINT | FIXED_JOINT | BODY | OP_FRAME | SENSOR )) const;
     
-    ///
-    /// \brief Get the name of a frame given by its index.
-    ///
-    /// \param[in] index Index of the frame.
-    ///
-    /// \return The name of the frame.
-    ///
-    PINOCCHIO_DEPRECATED const std::string & getFrameName (const FrameIndex index) const;
```

Comment:
I think it is relevant to remove that. It is a quite recent feature, and nobody should complain.